crates/gpui/src/platform/linux/x11/client.rs

```diff
@@ -343,21 +315,42 @@ impl Client for X11Client {
             .find(|m| m.id == mode_id)
             .expect("Missing screen mode for crtc specified mode id");
 
-        let refresh_millies = mode_refresh_rate_millis(mode);
-
-        self.refresh_millis.set(refresh_millies);
+        let refresh_event_token = self
+            .platform_inner
+            .loop_handle
+            .insert_source(calloop::timer::Timer::immediate(), {
```

Comment:
Does it makes sense to refresh every window at a different time? I thought X11 only supported a single refresh rate.

Reply:
The refresh rate is a property of the display on which the window is currently displayed